src/execution/__tests__/executor-test.js

```diff
@@ -393,6 +397,10 @@ describe('Execute: Handles basic execution tasks', () => {
         // eslint-disable-next-line no-throw-literal
         throw 'Error getting syncRawError';
       },
+      syncObjectError() {
```

Comment:
Since we don't distinguish between different types of non-Errors I don't think we need to add this test.